### ISSUE TITLE
Problems typing with both the eyes free keyboard and the defalt android keyboard

### ISSUE THREAD

**Issue by GoogleCodeExporter** [clc/eyes-free#76]
```
When typing it seems every third or fourth letter will cause either a letter to 
be entered before lifting you finger or in the case of the eyes free keyboard 
it will put to of the same charactor in.  This makes typing difficult.  What 
steps will reproduce the problem?
1. It could be touch and explore conflicting with typing but not sure
2.
3.

What is the expected output? What do you see instead?
I would like this to be addressed.  If it is down to touch and explore maybe 
when in an edit field for it to be disabled until a particular key stroke or 
geture was made may fix this

What version of the product are you using? On what operating system?
I am using the first release of ics on the nexus

Please provide any additional information below.


```

Original issue reported on code.google.com by `NeilMcCu...@gmail.com` on 2 Dec 2011 at 11:15

**Comment by GoogleCodeExporter** [comment]
```
Interference from touch exploration's handling of tap events.
```

Original comment by `alanv@google.com` on 6 Dec 2011 at 11:48
- Added labels: **Priority-High**
- Removed labels: **Priority-Medium**

**Comment by GoogleCodeExporter** [comment]
```
Resolved in v1.1.1 update.
```

Original comment by `alanv@google.com` on 17 Dec 2011 at 6:01
- Changed state: **Fixed**
